dev-tools/scripts/releaseWizard.yaml

```diff
@@ -617,8 +621,10 @@ groups:
     - https://cwiki.apache.org/confluence/display/SOLR/Release+Notes
   - !Todo
     id: new_jira_versions
-    title: Add a new version in JIRA for the next release
+    title: (PMC) Add a new version in JIRA for the next release
     description: |-
+      WARNING: Only PMC members have admin access to JIRA. If you are not a PMC member, please reach out to one to complete this step.
```

Comment:
Committers can easily be given elevated rights in jira. So perhaps instead of or in addition to the “ask a pmc member” notes, the preparation section write:

“If you are not a PMC member and this is your first time as RM, please ask to be granted extra permissions in Jira in order to complete all steps of the release.”